frontend/src/constants/resources.ts

```diff
@@ -73,7 +73,6 @@ export enum CONFIG_TITLE {
 }
 
 export const BOARD_TYPES = {
-  CLASSIC_JIRA: 'Classic Jira',
```

Comment:
are u sure?  we have plan our HB will not support `Classic Jira`, just yes or no?

Reply:
We still support `Classic Jira`. It's just that there is no need for the user to select it manually, but the backend acquires the type directly through API.